katharsis-core/src/main/java/io/katharsis/dispatcher/controller/resource/ResourceHead.java

```diff
@@ -0,0 +1,52 @@
+package io.katharsis.dispatcher.controller.resource;
+
+import io.katharsis.dispatcher.controller.HttpMethod;
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.repository.RepositoryMethodParameterProvider;
+import io.katharsis.request.dto.RequestBody;
+import io.katharsis.request.path.JsonPath;
+import io.katharsis.request.path.PathIds;
+import io.katharsis.request.path.ResourcePath;
+import io.katharsis.resource.exception.ResourceNotFoundException;
+import io.katharsis.resource.include.IncludeLookupSetter;
+import io.katharsis.resource.registry.RegistryEntry;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.resource.registry.RegistryEntry'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)